docs/rules/order.md

```diff
@@ -210,12 +212,14 @@ import React, { PureComponent } from 'react';
 import aTypes from 'prop-types';
 import { compose, apply } from 'xcompose';
 import * as classnames from 'classnames';
+import blist from 'BList';
```

Comment:
perhaps a relative path would be better here, since npm package names can't be capitalized (for many years now)